contracts/base/BaseLiquidityHandler.sol

```diff
@@ -0,0 +1,231 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
another high level design comment - its not actually really clear the split between these contracts anyways? BaseLiquidityManagement seems like it could actually be radically simplified (& combined with this contract) 

All of those functions in BaseLiquidityManagement just end up calling _modifyLiquidity on core.. so why not just expose external functions:

collect
add
remove

that then all just call the same _internal_ _modifyLiquidity function with the correct parameters that then are used to determine the proper encoding for the callback data?